Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -465,11 +577,12 @@ public struct TLSConfiguration {
                                 renegotiationSupport: .none,  // Servers never support renegotiation: there's no point.
                                 additionalTrustRoots: additionalTrustRoots)
     }
-    
+
     /// Creates a TLS configuration for use with client-side contexts. This allows setting the `NIOTLSCipher` property specifically.
     ///
     /// This provides sensible defaults, and can be used without customisation. For server-side
     /// contexts, you should use `forServer` instead.
+    @available(*, deprecated, renamed: "TLSConfiguration.forClient()")
     public static func forClient(cipherSuites: [NIOTLSCipher],
```

Comment:
Does deprecating these constructors move away from setting fields like `[NIOTLSCipher]` on `forClient` and `forServer` and move these customizations towards something like?
```swift
var client = TLSConfiguration.forClient()
client.cipherSuiteValues = [.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384]
```

Reply:
Yes, that's the goal. This avoids the painful process you had to go through of repeatedly adding new initialisers.

Reply:
Okay, thank you.